### PR TITLE
List the EBL models

### PR DESCRIPTION
**Pull request by bkhelifi**
**Description**

This simple pull request adds in the user guide how to get the list of EBL models .

### CODE REVIEW

File: examples/models/spectral/plot_absorbed.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 (`~gammapy.modeling.models.EBLAbsorptionNormSpectralModel`), which depends on the energy of the gamma-rays and the
 redshift z of the source, and :math:`\alpha` is a scale factor
 (default: 1) for the optical depth.
+
+The list of available EBL models can be found by printing `~gammapy.modeling.models.spectral.EBL_DATA_BUILTIN`.
```
Comment (QRemy) [inline examples/models/spectral/plot_absorbed.py:19]:
```suggestion

The available EBL models are defined in `~gammapy.modeling.models.spectral.EBL_DATA_BUILTIN`.
```

**Review by QRemy — COMMENTED** [review]
Thanks @bkhelifi, at the beginning of the example you could explicitly show :
```
from gammapy.modeling.models.spectral import EBL_DATA_BUILTIN
print(EBL_DATA_BUILTIN.keys())
```

**Comment by adonath** [comment]
> Thanks @bkhelifi, at the beginning of the example you could explicitly show :

Or use `pprint` instead...

**Review by AtreyeeS — APPROVED** [review]
Thanks @bkhelifi ! This is useful indeed.

As a follow up, I think it would be nice to have some information available on the models, at least a citation to the relevant paper. Maybe a `info` or a `meta` parameter on these models?

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi . Looks good.

**Pull request merged**